### PR TITLE
support: Add new EVMs to Support NFTs 

### PR DESCRIPTION
**Pull request by mcayuelas-ledger**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

1. Add new EVMs in NFTs support (https://github.com/LedgerHQ/nft-metadata-service/pull/1448)

`arbitrum, avalanche_c_chain, base, blast, bsc, celo, ethereum, fantom, linea, moonbeam, optimism, polygon, scroll`

LLD

Screenshot 2024-09-26 at 16 12 17
Screenshot 2024-09-26 at 15 36 05
Screenshot 2024-09-26 at 15 35 18

LLM

https://github.com/user-attachments/assets/aff94d50-4661-400e-9644-b6343350dd9a

https://github.com/user-attachments/assets/79f7d3ed-fcb2-4181-9b01-c1d1bcd3b6dc

2. Update SimpleHash tool in LLD to support more Spam report/check

Screenshot 2024-09-26 at 16 38 02

### ❓ Context

- **JIRA or GitHub link**: [LIVE-14272] 

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-14272]: https://ledgerhq.atlassian.net/browse/LIVE-14272?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by LucasWerey — APPROVED** [review]

**Comment by LucasWerey** [comment]
Tested and it works well. Looks like we don't receive rarible/opensea links from backend so all good

File: libs/env/src/env.ts [hunk 1]
```diff
@@ -537,8 +537,23 @@ const envDefinitions = {
...
+      "bsc",
+      "celo",
+      "ethereum",
+      "fantom",
+      "linea",
+      "moonbeam",
+      "optimism",
+      "polygon",
+      "scroll",
+      "sei_network",
```
Comment (lambertkevin) [inline libs/env/src/env.ts:554]: Activating NFTs on those chains might make the synchronization process ~2 times longer on explorers with rate limits. 
Please make sure it's considered acceptable by product that rate limited chains go from 15 to 25 seconds per sync (including discovery at the "addAccount" level).
Also I'm not sure we have anything in the coin-integration of cosmos for the support of NFTs, so `sei_network` might not work unless the whole portfolio is built frontend wise.

**Review by lambertkevin — DISMISSED** [review]
LGTM for the coin-integration parts. Important information to share with product though, and to share with the coin-integration product as well please 👍

File: libs/env/src/env.ts (hunk 1 above)
Comment (mcayuelas-ledger) [reply libs/env/src/env.ts:554]: Yep totally agree, I'll talk to Product :) 

thanks 🙏🏻

**Review by LucasWerey — APPROVED** [review]

**Pull request merged**